src/AssociationRegistry.Admin.Api/Magda/MagdaGeefVerenigingService.cs

```diff
@@ -66,11 +77,39 @@ public async Task<Result<VerenigingVolgensKbo>> GeefVereniging(KboNummer kboNumm
         }
     }
 
+    private static NaamOndernemingType? GetBestMatchingNaam(NaamOndernemingType[] namen)
+    {
+        var activeNamen = GetActiveNamen(namen);
+
+        if (activeNamen.Length == 0)
+            return null;
+        if (activeNamen.Length == 1)
+            return activeNamen.Single();
+
+        return GetNaamInTaal(activeNamen, TaalCodes.Nederlands) ??
+               GetNaamInTaal(activeNamen, TaalCodes.Frans) ??
+               GetNaamInTaal(activeNamen, TaalCodes.Duits) ??
+               GetNaamInTaal(activeNamen, TaalCodes.Engels);
```

Comment:
fallback: indien enkel  een of meerdere actieve in andere talen, 1e nemen daarvan